internal/pkg/cli/svc_deploy.go

```diff
@@ -342,12 +343,17 @@ func buildArgs(name, imageTag, copilotDir string, unmarshaledManifest interface{
 		tags = append(tags, imageTag)
 	}
 	args := mf.BuildArgs(filepath.Dir(copilotDir))
+	platform := mf.TaskPlatform()
```

Comment:
Do we also need these lines for `job deploy`?

Also, do you think the validation function could be invoked from `TaskPlatform`? So the function signature would be `func (t *TaskConfig) TaskPlatform() (string, error)` which returns an error if the platform is invalid.

Reply:
`job_deploy` uses the `buildArgs()` function in `svc_deploy`, but I'll manually test to double check.

I stuck the validation in `TaskPlatform()`! 👍🏼